internal/crypto/key_derivation_nontest.go

```diff
@@ -5,22 +5,37 @@ package crypto
 
 import (
 	"github.com/pkg/errors"
-	"golang.org/x/crypto/scrypt"
+)
+
+const (
+	// MasterKeyLength describes the length of the master key.
+	MasterKeyLength = 32
+
+	// ScryptAlgorithm is the key for the scrypt algorithm.
+	ScryptAlgorithm = "scrypt-65536-8-1"
+	// Pbkdf2Algorithm is the key for the pbkdf algorithm.
+	Pbkdf2Algorithm = "pbkdf2"
```

Comment:
The `pbkdf2` "algorithm name" is too generic. This should be more specific and include what parameters are being used. For example, this should be something along the lines of `pbkdf2-sha256-<num-iterations>` or even more specific depending on what parameters we allow to overwrite.


(Granted, the "algorithm" infix does not make for a good name for these constants since it is too generic, and thus does not precisely convey what the values for these constants are)

Reply:
Addressed in #3811